src/main/java/frc/trigon/robot/subsystems/turret/simulationturret/SimulationTurretConstants.java

```diff
@@ -0,0 +1,37 @@
+package frc.trigon.robot.subsystems.turret.simulationturret;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+
+public class SimulationTurretConstants {
+    static final double VOLTAGE_COMPENSATION_SATURATION = 12;
+
+    private static final double
+            MAX_MOTOR_VELOCITY = 200,
+            MAX_MOTOR_ACCELERATION = 3;
+    private static final TrapezoidProfile.Constraints CONSTRAINTS = new TrapezoidProfile.Constraints(MAX_MOTOR_VELOCITY, MAX_MOTOR_ACCELERATION);
```

Comment:
Put this after declaring the motor since this isn't a motor setting, and put this next to the profiled controller